source/base/bounding_box.cc

```diff
@@ -0,0 +1,43 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by Giovanni Alzetta
```

Comment:
I'm afraid that if you contribute the code to the project, it needs to be "the deal.II authors" here as in all other files :-(

Reply:
Yes, sorry...that comes from a too stupid copy-paste from the other files (which is also revealing about my degree as "author" )